gp-includes/install-upgrade.php

```diff
@@ -6,12 +6,26 @@
  * @since 1.0.0
  */
 
+/**
+ * Tells whether the database needs to be installed/upgraded or not.
+ *
+ * @since 4.0.0
+ */
+function gp_db_upgrade_required(): bool {
+	$gp_db_version = get_option( 'gp_db_version' );
+
+	return false === $gp_db_version || GP_DB_VERSION > $gp_db_version;
```

Comment:
If we don't have  `gp_db_version` in the database, `get_option( 'gp_db_version' )` will return `false`, so the `GP_DB_VERSION > get_option( 'gp_db_version' )` will be `true`. It will happen the same if `gp_db_version` is less than `GP_DB_VERSION`, so I suggest this change:

```suggestion
	return GP_DB_VERSION > get_option( 'gp_db_version' );
```

Reply:
I didn't realise that `GP_DB_VERSION > false` would be `true` 👍